.github/workflows/awstest.yml

```diff
@@ -6,11 +6,13 @@ on:
   push:
     branches:
       - master
+      - dev
```

Comment:
You're doing AWS tests on `dev`?

Reply:
Ah, I literally just added that in the last commit to `dev` to test if the pipeline is working on AWS. Have removed it already.